src/ui/common/src/components/pages/AccountPage.tsx

```diff
@@ -49,7 +49,13 @@ client = aqueduct.Client(
         <Typography variant="body1" sx={{ fontWeight: 'bold', mr: '8px' }}>
           Python SDK Connection Snippet
         </Typography>
-        <CodeBlock language="python">{apiConnectionSnippet}</CodeBlock>
+        <CopyBlock
+          text={apiConnectionSnippet}
+          language="python"
+          showLineNumbers={false}
+          theme={monoBlue}
+          highlight="1-5"
```

Comment:
What does this mean?

Reply:
Specifically the highlight line

Reply:
<img width="860" alt="image" src="https://user-images.githubusercontent.com/61630725/175358697-dc2ea3a1-2d52-4fdd-9c24-1a6566ca15cd.png">
https://github.com/rajinwonderland/react-code-blocks

Reply:
What is the difference between highlight and not highlighted? Curious because the snippet doesn't seem like it is highlighting anything.